### ISSUE TITLE
Dependency problems,

### ISSUE THREAD

**Issue by EndrII** [DeadAirRT/deadairware#1]
It seems that this may have dependency problems, when I plug it in, I get files from other mods
on Enhancer - Econ Ai Module and Enhancer - Econ Balance. And now I'm not sure about the performance of my game.
![20200505191755_1](https://user-images.githubusercontent.com/12465465/81090088-e0f54400-8f05-11ea-9545-1acedbe1cbb0.jpg)

**Comment by DeadAirRT** [comment]
They are supposed to be optional, it should not prevent loading. Can you check content.xml and see if steam is breaking the dependency?

**Comment by EndrII** [comment]
``` xml
<?xml version="1.0" encoding="utf-8"?>
<content id="ws_2082610782" name="DeadAir Ware" description="DeadAir Ware&#13;&#10;&#13;&#10;Scales production cycles to 5 minutes or less&#13;&#10;- Increases critical ship ware production amounts&#13;&#10;- Increases workforce effect to 100% and growthrate&#13;&#10;- Removes faction relation discounts to prevent unsellable goods&#13;&#10;- Lowers station module build time&#13;&#10;- Workforce now requires energy cells as well&#13;&#10;&#13;&#10;Save game compatible.&#13;&#10;&#13;&#10;Patch notes available on:&#13;&#10;https://github.com/DeadAirRT&#13;&#10;&#13;&#10;Best place to contact me:&#13;&#10;https://forum.egosoft.com/viewtopic.php?f=181&amp;t=415937" author="DeadAir" version="118" date="2020-05-02" save="0" lastupdate="1588466460">
  <dependency id="ego_dlc_split" optional="false" name="Split Vendetta"></dependency>
  <dependency id="ws_1668457361" optional="true" name="Faction Enhancer - War Module"></dependency>
  <dependency id="ws_1668473070" optional="true" name="Faction Enhancer - Econ AI Module"></dependency>
  <dependency id="ws_1668472321" optional="true" name="Faction Enhancer - Econ Balance Module"></dependency>
  <text language="7" name="DeadAir Ware" description="Adjust Production Amount and Volume." author="DeadAir"></text>
  <text language="33" name="DeadAir Ware" description="Adjust Production Amount and Volume." author="DeadAir"></text>
  <text language="34" name="DeadAir Ware" description="Adjust Production Amount and Volume." author="DeadAir"></text>
  <text language="39" name="DeadAir Ware" description="Adjust Production Amount and Volume." author="DeadAir"></text>
  <text language="44" name="DeadAir Ware" description="Adjust Production Amount and Volume." author="DeadAir"></text>
  <text language="45" name="DeadAir Ware" description="Adjust Production Amount and Volume." author="DeadAir"></text>
  <text language="49" name="DeadAir Ware" description="Adjust Production Amount and Volume." author="DeadAir"></text>
  <text language="81" name="DeadAir Ware" description="Adjust Production Amount and Volume." author="DeadAir"></text>
  <text language="82" name="DeadAir Ware" description="Adjust Production Amount and Volume." author="DeadAir"></text>
  <text language="86" name="DeadAir Ware" description="Adjust Production Amount and Volume." author="DeadAir"></text>
  <text language="88" name="DeadAir Ware" description="Adjust Production Amount and Volume." author="DeadAir"></text>
</content>
```
After the 2nd launch all good
I think this is a game problem.
